Tools/docker_run.sh

```diff
@@ -5,14 +5,11 @@ if [ -z ${PX4_DOCKER_REPO+x} ]; then
 	if [[ $@ =~ .*px4_fmu.* ]]; then
 		# nuttx-px4fmu-v{1,2,3,4,5}
 		PX4_DOCKER_REPO="px4io/px4-dev-nuttx:2019-10-24"
-	elif [[ $@ =~ .*navio2.* ]] || [[ $@ =~ .*raspberry.* ]] || [[ $@ =~ .*bebop.* ]]; then
-		# posix_rpi_cross, posix_bebop_default
-		PX4_DOCKER_REPO="px4io/px4-dev-raspi:2019-10-24"
 	elif [[ $@ =~ .*eagle.* ]] || [[ $@ =~ .*excelsior.* ]]; then
 		# eagle, excelsior
 		PX4_DOCKER_REPO="lorenzmeier/px4-dev-snapdragon:2018-09-12"
-	elif [[ $@ =~ .*ocpoc.* ]]; then
-		# aerotennaocpoc_ubuntu
+	elif [[ $@ =~ .*ocpoc.* ]] || [[ $@ =~ .*navio2.* ]] || [[ $@ =~ .*raspberry.* ]] || [[ $@ =~ .*bebop.* ]]; then
```

Comment:
Bebop won't work with the new toolchain but it's probably broken anyway at this point.

Reply:
It was likely going to be dropped with DriverFramework anyway. Do you know why it wouldn't work?

Reply:
https://github.com/PX4/Firmware/issues/9807